apiserver/authentication/agent.go

```diff
@@ -13,9 +13,9 @@ import (
 )
 
 // EntityAuthenticator performs authentication for juju entities.
-type EntityAuthenticator struct{}
+type AgentAuthenticator struct{}
```

Comment:
Why the rename? 

Comment is now wrong.

Reply:
Because we naming conflicts with some of the new duck typing so was easier to rename this to what it was actually authenticating.

Specifically entity is a loose term that could mean either an actual user or an agent and so on. Where this strategy is specifically just dealing with agents.

The interface that this implements is geared towards entities as a whole where this struct is for a specific subset. It is the whole is a car a subset of vehicle argument.